### PR TITLE
Enh: ansible roles for router (c4)

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: roles/router/templates/kamailio-local.cfg.j2 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* add API address used for routing */
+#!define HTTP_API_ENDPOINT "{{ router_api_endpoint_confd }}/kamailio/routing"
+
+/* add API http timeout */
+#!define HTTP_API_TIMEOUT 5000
+
+/* add listen protocol, address and port / advertise (optional) */
+{% if router_advertise_address is defined and router_advertise_address %}
+#!define LISTEN udp:{{ ansible_default_ipv4.interface }}:5060 advertise {{ router_advertise_address }}
```
Comment (sboily) [inline roles/router/templates/kamailio-local.cfg.j2]: You need to add a port for the advertise. ip:port. It would be nice to have it in variable.

**Review by fredericlepied — DISMISSED** [review]

File: zuul.yaml [hunk 2]
```diff
@@ -1,20 +1,32 @@
 - job:
     parent: base
-    name: wazo-ansible-install
+    name: wazo-ansible-uc
     required-projects:
       wazo-platform/wazo-acceptance
-    description: install Wazo platform
+    description: install Wazo Platform UC user case
```
Comment (sboily) [inline zuul.yaml]: user to use?

**Review by sboily — COMMENTED** [review]
For me it works, but the eth0 hardcoded is not good, and we are using the kamailio repo, i would like to only have the wazo repo with the kamailio version we checked.

File: zuul.yaml (hunk 2 above)
Comment (fredericlepied) [reply zuul.yaml]: fixed

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
@sboily can we do that (eth0 hardcoded and kamailio repo) in another PR or do you want to fix that in this one?

**Comment by fredericlepied** [comment]
recheck

**Review by sboily — APPROVED** [review]

**Pull request merged**